server/mysterium_api.go

```diff
@@ -77,6 +77,8 @@ func (mApi *mysteriumApi) NodeSendStats(nodeKey string, signer identity.Signer)
 	err = mApi.doRequest(req)
 	if err == nil {
 		log.Info(mysteriumApiLogPrefix, "Node stats sent: ", nodeKey)
+	} else {
+		log.Error(mysteriumApiLogPrefix, "Node stats failed: ", err)
```

Comment:
Before `client_rest.go` had common error loging, which was removed during refactoring.
Can You return it back:
```
func (client *mysteriumRestClient) executeRequest(method, fullPath string, payloadJson []byte) (*http.Response, error) {
	request, err := http.NewRequest(method, fullPath, bytes.NewBuffer(payloadJson))
	request.Header.Set("User-Agent", MYSTERIUM_API_CLIENT)
	request.Header.Set("Content-Type", "application/json")
	request.Header.Set("Accept", "application/json")
	if err != nil {
		log.Critical(MYSTERIUM_API_LOG_PREFIX, err)
		return nil, err
	}

	response, err := client.http.Do(request)

	if err != nil {
		log.Error(MYSTERIUM_API_LOG_PREFIX, err)
		return response, err
	}

	err = parseResponseError(response)
	if err != nil {
		log.Error(MYSTERIUM_API_LOG_PREFIX, err)
		return response, err
	}

	return response, nil
}
```

Reply:
Done.